pkg/client/openstack/client.go

```diff
@@ -536,7 +551,22 @@ func (c *client) GetNodes(kluster *kubernikus_v1.Kluster, pool *models.NodePool)
 	return nodes, nil
 }
 
-func (c *client) CreateNode(kluster *kubernikus_v1.Kluster, pool *models.NodePool, userData []byte) (string, error) {
+func (c *client) CreateNode(kluster *kubernikus_v1.Kluster, pool *models.NodePool, userData []byte) (id string, err error) {
+	var name string
+
+	defer func() {
+		c.Logger.Log(
+			"msg", "created node",
+			"kluster", kluster.Name,
+			"project", kluster.Account(),
+			"name", name,
+			"id", id,
+			"err", err,
```

Comment:
the `TrailingNilFilter` will not remove this because of the following `v` key. If we add an `err` field that might be nil, it should be the last key.

Reply:
Ah, I didn't know it works like that. I thought it just checks for `err ==  nil`